### PR TITLE
Implement sound meter in the SPN and worklet recorder

### PR DESCRIPTION
**Pull request by TenghuiZhang**
This PR fix issue #332

### CODE REVIEW

**Comment by TenghuiZhang** [comment]
Current UI: 
volumn-meter

**Comment by TenghuiZhang** [comment]
This is the screenshot of vm-meter with 7px wide
vm-meter

File: src/audio-worklet/migration/spn-recorder/app.js [hunk 1]
```diff
@@ -11,29 +11,34 @@ const context = new AudioContext();
 // Arbitrary buffer size, not specific for a reason
 const BUFFER_SIZE = 256;
 // Make the visulization more clear to the users
-const WAVEFROM_SCALE_FACTOR = 5
+const WAVEFROM_SCALE_FACTOR = 5;
+// Make the visulization of vu meter more clear to the users
```
Comment (hoch) [inline src/audio-worklet/migration/spn-recorder/app.js:15]: nit: "more clear" -> "clearer". A missing period at the end of the sentence.

File: src/audio-worklet/migration/spn-recorder/app.js [hunk 2]
```diff
@@ -118,10 +120,9 @@ function setupScriptProcessor(recordingProperties, passSampleToVisualizers) {
     // Update tracked recording length.
     if (isRecording) {
       recordingLength += BUFFER_SIZE;
-      
       if (recordingLength > recordingProperties.maxFrameCount) {
         isRecording = !isRecording;
-
+        recordText.textContent = 'Reach the maximum length of';
```
Comment (hoch) [inline src/audio-worklet/migration/spn-recorder/app.js:125]: Perhaps "Reached" is correct.

File: src/audio-worklet/migration/spn-recorder/index.njk [hunk 3]
```diff
@@ -31,43 +85,35 @@ to_root_dir: ../../../
...
+      <button disabled id="stop">
+        Stop Recording
+      </button>
+      <a href="" id="download-link">
+        <button disabled id="download-button">Download File</button>
+      </a>
     </div>
 
-    <canvas id="recording-canvas"></canvas>
+    <div class="recording-display">
```
Comment (hoch) [inline src/audio-worklet/migration/spn-recorder/index.njk:103]: Thanks for trying this idea out, but I think we need to implement the VU meter with `canvas`.

**Review by hoch — COMMENTED** [review]
Can we create a reusable JS class for a volume meter? Then you don't have to repeat it in two different pages.

File: src/audio-worklet/migration/spn-recorder/index.njk (hunk 3 above)
Comment (TenghuiZhang) [reply src/audio-worklet/migration/spn-recorder/index.njk:103]: Sure, I open another PR which is using the canvas to design the vu-meter

**Comment by hoch** [comment]
Do we still need to pursue this? I think #335 is the one we need to review and merge.

**Pull request closed without merging**